### PR TITLE
docs(vue-router-url-sync): bootstrap page

### PR DESCRIPTION
**Pull request by rayrutjes**

### CODE REVIEW

File: docs/docgen/src/advanced/vue-router-url-sync.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+mainTitle: Advanced
+layout: main.pug
+category: Advanced
+withHeadings: true
+navWeight: 3
+editable: true
+githubSource: docs/docgen/src/advanced/vue-router-url-sync.md
+---
+
+In this guide, you will learn how to synchronize your InstantSearch experience with the [official Vue-Router](https://github.com/vuejs/vue-router).
```
Comment (Haroenv) [inline docs/docgen/src/advanced/vue-router-url-sync.md]: no comma here

File: docs/docgen/src/advanced/vue-router-url-sync.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+## Example app
+
+A fully working example of synchronizing the Vue router with Vue InstantSearch can be found here: https://github.com/algolia/vue-instantsearch-examples/tree/master/examples/vue-router
+
+## Initialize InstantSearch with route parameters
+
+In the example app linked above, we register one route that looks like this:
+
+```javascript
+{
```
Comment (Haroenv) [inline docs/docgen/src/advanced/vue-router-url-sync.md]: might be clearer if it's the complete call: 

```js
const router = new VueRouter({
  routes: [
    {
      name: 'search',
      path: '/search',
      component: Search,
      props: route => ({ query: route.query.q }),
    },
    { path: '/', redirect: '/search' },
  ],
});
```

File: docs/docgen/src/advanced/vue-router-url-sync.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    <!-- ... -->
+  </ais-index>
+</template>
+```
+
+Now, every time the route changes, the search store query will be updated with the value coming from the URL `q` query parameter.
+
+## Keep Vue router in sync with Vue router
+
+Previous section helped us making sure that when a route changes, the search state is also updated.
```
Comment (Haroenv) [inline docs/docgen/src/advanced/vue-router-url-sync.md]: The previous section made our search state aware of the route, here we will learn ...

File: docs/docgen/src/advanced/vue-router-url-sync.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+Here you will learn how to push new state to the router in order to keep URL's in sync with the current search state as users interact with it.
```
Comment (Haroenv) [inline docs/docgen/src/advanced/vue-router-url-sync.md]: URLs

**Comment by rayrutjes** [comment]
I integrated all your feedback @Haroenv , thanks!

**Pull request merged**